tests/expectations/test_expectation.py

```diff
@@ -466,6 +466,19 @@ def test_expectation_equality_ignores_rendered_content():
     assert expectation_a == expectation_b
 
 
+@pytest.mark.unit
+def test_expectation_with_row_condition_generates_rendered_content():
+    expectation = gxe.ExpectColumnValuesToBeBetween(
+        column="foo",
+        min_value=0,
+        max_value=10,
+        row_condition=Column(name="status") == "active",
+        condition_parser="great_expectations",
+    )
+    expectation.render()
+    assert expectation.rendered_content is not None
```

Comment:
Seems like the implementation is probably right, but I think `_get_row_condition_string` was doing some string replacement that we wouldn't want. Can you just add an assert here on the actual string it's set to? Probably just `== repr(Column(name="status") == "active")`?